.github/workflows/actions.yml

```
@@ -93,7 +93,8 @@ jobs:
                keras_cv/models/backbones \
                keras_cv/models/classification \
                keras_cv/models/object_detection/retinanet \
-               --durations 0
+               keras_cv/models/object_detection/yolo_v8 \
+               --durations 0 --run_large
```

`--run_large` flag is already set.

```suggestion
               --durations 0
```

---

Maybe just missing a merge from `core` in your feature branch?